src/Logbook/ILogbook.sol

```diff
@@ -13,7 +13,7 @@ interface ILogbook is IRoyalty, IERC721 {
     error InvalidBPS(uint256 min, uint256 max);
     error InvalidTokenId(uint256 min, uint256 max);
     error InsufficientAmount(uint256 available, uint256 required);
-    error InsufficientLogs(uint256 logCount);
+    error InsufficientLogs(uint32 logCount);
```

Comment:
Does uint in error need to be packed with other variables as well?

Reply:
Solidity seems only pack storage variables https://docs.soliditylang.org/en/latest/internals/layout_in_storage.html